config/confighttp/confighttp.go

```diff
@@ -85,6 +111,16 @@ func (hcs *HTTPClientSettings) ToClient(ext map[config.ComponentID]component.Ext
 		}
 	}
 
+	// Compress the body using specified compression methods if non-empty string is provided.
+	// Supporting Gzip, zlib, snappy, and zstd.
```

Comment:
should this include `deflate` as well in this list?

Reply:
fixed